README.md

```diff
@@ -1,5 +1,7 @@
 # best-restaurants
-Shows 20 restaurants with best ratings from Wroclaw
+Shows 20 restaurants with best ratings from Wroclaw.
+The application bases on the rating from zomatoAPI.
```

Comment:
"ZamatoAPI", or more precisely: the fact that your way to get the Zomato data is via REST API, is an implementation detail. "Zomato" is enough.